sdk/storage/Azure.Storage.Blobs/tests/BlockBlobClientTests.cs

```diff
@@ -296,6 +296,44 @@ public async Task StageBlockAsync_Error()
             }
         }
 
+        [LiveOnly]
```

Comment:
Does this need to be LiveOnly?

Reply:
It transfers 100 MB of data, I don't think we want that in our test files.

Reply:
This test is only 1KB.